### ISSUE TITLE
Any interest in making an option to mute a thread?

### ISSUE THREAD

**Issue by nirmit** [nylas-mail-lives/nylas-mail#175]
I wanted to know from the community:
1.  Do you want an option tpo

**Comment by dweremeichik** [comment]
Ha ha ha, this would be great to battle those annoying co-workers who can't figure out how to use "Reply" instead of "Reply all". I like it :+1:

**Comment by nirmit** [comment]
Can we make a plug-in that users can download and install, or would you like the idea to merge it with unsubscribe?

**Comment by seesemichaelj** [comment]
I don't think it should be merged with the unsubscribe plugin; from a technical perspective, they're too different. A plugin would be nice; we can decide at a later time (if the plugin exists) whether or not it's built in.
